src/geoaxis.jl

```diff
@@ -104,3 +102,27 @@ function GeoAxis(args...;
     hidespines && hidespines!(ax)
     return ax
 end
+
+
+function Makie.apply_transform(f::Proj4.Transformation, pt::Point)
+    Point(f(Vec(pt)) ./ 10_000f0)
```

Comment:
I see this constant is now 10x smaller. Do we even need it at all? Is this to get smaller `eps()` with big coordinates? We don't know in general if the input coordinates will be ~1e2 or ~1e5.

Reply:
Ah, I didn't notice, that was my mistake.
By default, proj gives [output units in metres](https://stackoverflow.com/a/22731202/392585), so my guess is that it is simply to give smaller magnitude values: @SimonDanisch is there any reason why this was needed?

Reply:
Would be good to know. If there is no good reason for it, it would be easier to transform according to the transformation given, without extra scaling.

I tested it and it seems to work fine without this.

Reply:
Changing this back to 100_000 fixed the tick labels.

Reply:
Oh lol then that must be the reason. Seems like that number is hiding the underlying issue. But no need to address that in this PR as well. Also 100_000 is a bit off:

![image](https://user-images.githubusercontent.com/4471859/157974578-62e81846-94fe-4fe5-8552-aa057a251fd3.png)

With bent meridians placing ticks below the plot is difficult in general, for instance if they all come together at the poles.